tests/testthat/test-join_keys-c.R

```diff
@@ -118,17 +118,16 @@ testthat::test_that("c.join_key_set merges with empty and non-empty parents", {
   jk2 <- join_keys(
     join_key("d3", "d3", "c"),
     join_key("d4", "d4", "d"),
-    join_key("d4", "d3", "cd")
+    join_key("d3", "d4", "cd")
   )
-  parents(jk2) <- list(d3 = "d4")
 
   expected <- join_keys(
     join_key("d1", "d1", "a"),
     join_key("d3", "d3", "c"),
     join_key("d4", "d4", "d"),
-    join_key("d3", "d4", "cd")
+    join_key("d3", "d4", "cd", directed = FALSE)
   )
-  parents(expected) <- list(d3 = "d4")
+  parents(expected) <- list(d4 = "d3")
```

Comment:
test for expect equal is reduntant. When c(jk1, jk2) is identical to expected it is certain that next testthat is true.

Reply:
It is. The test expanded the scope to test if both merges are equal. Removed!